modules/log6x-log-collector-http-server.adoc

```diff
@@ -0,0 +1,117 @@
+//
+// * observability/logging/log_collection_forwarding/cluster-logging-collector.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="log6x-log-collector-http-server_{context}"]
+= Configuring the collector to receive audit logs as an HTTP server
+
+You can configure your log collector to listen for HTTP connections to only receive audit logs by specifying `http` as a receiver input in the `ClusterLogForwarder` custom resource (CR).
+
+:feature-name: HTTP receiver input
+include::snippets/logging-http-sys-input-support.adoc[]
+
+
+.Prerequisites
+
+* You have administrator permissions.
+* You have installed the {oc-first}.
+* You have installed the {clo}.
+* You have created a `ClusterLogForwarder` CR.
+
+.Procedure
+
+. Modify the `ClusterLogForwarder` CR to add configuration for the `http` receiver input:
++
+--
+.Example `ClusterLogForwarder` CR
+[source,yaml]
+----
+apiVersion: observability.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+# ...
+spec:
+  inputs:
+  - name: http-receiver # <1>
+    type: receiver
+    receiver:
+      type: http # <2>
+      port: 8443 # <3>
+      http:
+        format: kubeAPIAudit # <4>
+  outputs:
+  - name: default-lokistack
+    lokiStack:
+      authentication:
+        token:
+          from: serviceAccount
+      target:
+        name: logging-loki
+        namespace: openshift-logging
+    tls:
+      ca:
+        key: service-ca.crt
+        configMapName: openshift-service-ca.crt
+    type: lokiStack
+# ...
+  pipelines: # <5>
+    - name: http-pipeline
+      inputRefs:
+        - http-receiver
+      outputRefs:
+        - <output_name>
+# ...
+----
+<1> Specify a name for your input receiver.
+<2> Specify the input receiver type as `http`.
+<3> Optional: Specify the port that the input receiver listens on. This must be a value between `1024` and `65535`. The default value is `8443` if this is not specified.
+<4> Currently, only the `kube-apiserver` webhook format is supported for `http` input receivers.
+<5> Configure a pipeline for your input receiver.
+--
+
+. Apply the changes to the `ClusterLogForwarder` CR by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f <filename>.yaml
+----
+
+.Verification
+
+. Verify that the collector is listening on the service that has a name in the format `<ClusterLogForwarder_CR_name>-<input_name>`. In this example, the service name is `collector-http-receiver`:
++
+[source,terminal]
+----
+$ oc get svc
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME                      TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)            AGE
+collector                 ClusterIP   172.30.85.239    <none>        24231/TCP          3m6s
+collector-http-receiver   ClusterIP   172.30.205.160   <none>        8443/TCP           3m6s
+----
+
+. Extract the certificate authority (CA) certificate file:
++
+[source,terminal]
+----
+$ oc extract cm/openshift-service-ca.crt -n <namespace>
+----
+
+. Send logs by using the `curl` command:
```

Comment:
```suggestion
. Use the `curl` command to send logs by running the following command:
```

- https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#procedure-step-that-introduces-a-command
> ...make sure that the step text includes an explicit instruction to "run" or "enter" the command.